api/types/database.go

```diff
@@ -1149,6 +1172,8 @@ const (
 	DatabaseTypeOpenSearch = "opensearch"
 	// DatabaseTypeMongoAtlas
 	DatabaseTypeMongoAtlas = "mongo-atlas"
+	// DatabaseTypeDocumentDB is the database type for AWS-hosted DocumentDB.
+	DatabaseTypeDocumentDB = "docdb"
```

Comment:
Does `docdb` is a official shortcut  cut for document db ? if not I would suggest using full name.

Reply:
Yes.
- CLI: https://docs.aws.amazon.com/cli/latest/reference/docdb/
- SDK: https://github.com/aws/aws-sdk-go-v2/tree/main/service/docdb